tools/sdk-cli/Cargo.lock

```diff
@@ -2360,9 +2360,9 @@ checksum = "e5ea92a5b6195c6ef2a0295ea818b312502c6fc94dde986c5553242e18fd4ce2"
 
 [[package]]
 name = "reqwest"
-version = "0.11.18"
+version = "0.11.23"
```

Comment:
How does this affect the Swift binding issue?

Reply:
Just to be sure, I rebased on `main`, it's using `0.11.20` now. Good catch.